### DISCUSSION TITLE
Rename the caption of the photo I uploaded for no reason

### DISCUSSION THREAD

**Discussion by Gpan666** [LycheeOrg/Lychee#2758] [Bugs]
### Lychee version

v6.1.2

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

When you upload photos with a title order, the uploaded album will be renamed to cosplaytele.com
![PixPin_2024-11-28_20-47-06](https://github.com/user-attachments/assets/5abb65ac-c01c-49f2-a157-94bde979ae66)
![QQ20241128-204902](https://github.com/user-attachments/assets/bf45a21a-1172-40ba-8978-0572908197d9)

### Diagnostics [REQUIRED]

info
Latest version of PHP is 8.3
info
git (software) is not available.
info
Found 1 small2x that could be generated.
info
Found 2 placeholder images that have not been encoded.
info
Have you considered supporting Lychee? :)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment]
> When you upload photos with a title order, the uploaded album will be renamed to cosplaytele.com

I am not sure I understand, but I think it might also be because of the exif data ? 
Can you check those or...
I please send me a sample picture that is breaking to my mail (preferably not of the above mentioned lady): benoit@viguier.nl
